upp-neo4j-provisioner/cloudformation/jumpbox.yaml

```diff
@@ -10,7 +10,14 @@ Metadata:
         default: Network Configuration
       Parameters:
       - VPC
-      - SubnetId
```

Comment:
Is this just tidying up an unused parameter? Can't see it referenced anywhere else in the template. :)

Reply:
Indeed it is an old param, and I added the ones that are used.